src/roiextractors/testing.py

```diff
@@ -11,6 +11,29 @@
 inttype = (int, np.integer)
 
 
+import numpy as np
+
+from roiextractors import NumpyImagingExtractor
+
+
+def generate_dummy_imaging_extractor(
+    num_frames: int = 100,
```

Comment:
In the spirit of keeping an instance of an object generated via this method using only defaults as minimal as possible in order to improve testing speeds, what would you think of `30` frames instead? That's 1 full second of imaging sampled at 30Hz (somewhat average for this).

I know you can of course always override this default, but having to do so is an extra step that is easy to forget and I can't think of many cases that would require a longer video for testing purposes.

Reply:
Totally onboard with this. Changing.